content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,72 @@
+---
+title:       "ReactOS in 2020"
+author:      "Extravert34"
+date:        2020-12-28
+---
+
+Despite all the turbulence, we've got quite a rich year for ReactOS.
+Many bugs and instabilities were resolved, many more have been introduced. This year we hired two kernel developers full-time, this happens for the first time on the project history.
+
+The post highlights some of the changes which may may be interesting to the community.
+
+## Shell changes
+
+Shell doesn't see much attention recently, due to most of the work concentrated in the kernel, but there are still some useful fixes and feature implementations.
+
+* Katayama Hirofumi MZ made a great job implementing the Filesystem change notification feature (aka `SHChangeNotify`) in the shell. This is required for all applications monitoring the filesystem changes
+* Visual and usability improvements: "Size on disk" label in a file Properties dialog, key accelerators in Open/Save dialog and many more
+
+## RAPPS (a GSoC project)
+
+This is one of our three GSoC projects of the year. He Yang continues the work from previous GSoC years, by improving our RAPPS application manager. This includes UI & usability enhancements, showing app screenshots, command line scripting support.
+
+*picture*
+
+## Compiler and tooling upgrade
+
+The aging RosBE 2.1 toolset has served us well since 2014, but it's time to move on and incorporate all the useful features implemented in modern compilers and build tools.
+We now have GCC 8.4, CMake 3.17 as our defaults in the new RosBE 2.2, while maintaining compatibility with the latest CMake. The transition didn't happen without flaws though, a lot of effort was put by Colin Finck to make modern CMake work on Windows 2003 (and thus on ReactOS because we maintain compatibility with it). All the patches are freely available in our GitHub.
+
+Clang build is resurrected on the CI, with plans on supporting this compiler along with GCC and MSVC. ReactOS buils with Clang now, but still cannot boot though.
+
+A controversial decision was made to drop support of old MSVC compilers, and raise the C language standard to C99 for better cross-compiler compatibility. The deprecation brought up a big discussion over ML and GitHub.
+Still, this opens a window for future C++ standard library upgrades (maybe in 2021, who knows) and the support of modern C++ in our codebase.
+
+## Kernel and drivers
+
+As there are two developers working full-time in this field, there are quite some long-awaited changes:
+
+**Merge of the PnP storage stack** The new storage stack, derived from open-source Microsoft drivers came to replace the old NT4-compatible ReactOS storage stack. For the long time many workarounds were applied to make NT4 storage stack work in the NT5.2 environment. This era ends now, with a major step forward in compatibility with modern Windows, along with new features:
```

Comment:
```suggestion
**Merge of the PnP storage stack** The new storage stack, derived from open-source Microsoft drivers came to replace the old NT4-compatible ReactOS storage stack.
For a long time many workarounds were applied to make the NT4 storage stack work in the NT5.2 environment.
This era ends now, with a major step forward in compatibility with modern Windows, along with new features:
```

Reply:
There is not a clear enough separation between the "titles" of the subparagraphs, and the text of these.
Maybe put the text of the paragraph on a newline below the "title"?

Reply:
You checked this in the page or the PR?
Because markdown renders quite different

Reply:
I selected "view file" in order to have the markdown being rendered.